### PR TITLE
ci: add and require testing under 3.7

### PR DESCRIPTION
**Pull request by cclauss**
@joshuarli  Python 3.7 fails on https://travis-ci.com/getsentry/sentry-flake8/jobs/241214324#L273

### CODE REVIEW

**Comment by joshuarli** [comment]
Thanks for setting this up, I'll have a look at this soon.

**Comment by untitaker** [comment]
@joshuarli can you review this and cut a new version to unblock https://github.com/getsentry/sentry/pull/14906?

**Comment by joshuarli** [comment]
Yeah, although I'll need to get this working on 3.7: https://github.com/getsentry/sentry-flake8/issues/6#issuecomment-544680929. Not gonna worry about 3.8 right now.

File: setup.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
     author="Sentry",
     author_email="hello@sentry.io",
     install_requires=["flake8>=3.7.0,<3.8.0"],
-    test_requires=["pytest"],
+    tests_requires=["pytest"],
```
Comment (cclauss) [inline setup.py]: tests (plural) require (singular) https://setuptools.readthedocs.io/en/latest/setuptools.html

Comment (joshuarli) [reply setup.py]: Yeah haha, just noticed that. Gonna force push over.

**Comment by joshuarli** [comment]
Alright, thanks @cclauss and @untitaker!

**Pull request merged**